src/features/dsComponents/structure/FormatConfigHistory.tsx

```diff
@@ -13,11 +13,11 @@ const FormatConfigHistory: React.FunctionComponent<FormatConfigHistoryProps> = (
   const format = structure.format
   const formatConfig = structure.formatConfig
```

Comment:
rename this component to just `FormatConfig`, add a `showTrueOptionsOnly` prop, & use it when calling from history. This component is super old 😄